src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -31,8 +46,11 @@ public static String getSupportTeamEmailAddress(InternetAddress systemAddress) {
         return systemAddress.getAddress();
     }
 
-    public static String getContactHeader(InternetAddress systemAddress, String rootDataverseName) {
-        return BundleUtil.getStringFromBundle("contact.header", Arrays.asList(getSupportTeamName(systemAddress, rootDataverseName)));
+    public static String getContactHeader(InternetAddress systemAddress) {
+        return BundleUtil.getStringFromBundle("contact.header", Arrays.asList(getSupportTeamName(systemAddress)));
     }
 
+    public static void injectDataverseService(DataverseServiceBean dataverseService) {
```

Comment:
This is bad style IMHO. If this is to be a bean, add a constructor to have explicit dependency injection in a non-static field. At the bare minimum, this should not be a `public` function (IMHO package private, as used solely for testing). Or, as the field is package private already, tests could just set the field directly.